### PR TITLE
Release 26

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-    end
   end
 
   def newsletter
     authorize! :manage, :all
     @members = Member.confirmed.wants_newsletter.all
-    respond_to do |format|
-      format.html # index.html.haml
-    end
+    respond_with @member
```
Comment (Br3nda) [inline app/controllers/admin_controller.rb]: Looks like a bug we missed. Better fix (and add regression tests) or newletter will break

Comment (Br3nda) [inline app/controllers/admin_controller.rb]: fixed in #1352

**Comment by Br3nda** [comment]
Need #1341 to pass review, then we can release this.

**Comment by cesy** [comment]
Why is this dependent on that? It would be good to get it live either way.

**Comment by Br3nda** [comment]
I have deleted my account on staging.
My photos are gone -- garden, plantings, harvest. The forum post and comments

While still logged in (before deletion) i discovered i can't comment on old forum posts. That's a separate problem. Seems forum commenting is broken on really old data.

**Comment by Br3nda** [comment]
If you start a forum thread, the whole thread will 404
http://www.staging.growstuff.org/posts/brenda-20170611-what

**Comment by cesy** [comment]
Okay, let's figure out forums separately, then, but looks like this is good to go.

**Pull request merged**